javascript/javascript3/week2/currency_converter/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Currency converter - JavaScript 3 week 2 homework</title>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <h1>Currency converter</h1>
+      <input type="number" name="base-amount" id="amount" value="1" />
+      <label for="base">from: </label
+      ><select name="base-currency" id="base">
+        <option value="EUR">EUR</option>
+      </select>
+      <label for="target">to: </label
+      ><select name="target-currency" id="target"></select>
```

Comment:
Please format your code - Prettier can help there https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
I actually use prettier from the day one at HYF - maybe there is a difference in its config. Mine config shows closing bracket ">" at the beginning of the next line in certain cases... I don't know if its important, but I can chang it if necessary